app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from typing import Any
+
+
+def calculate_team_total_rating(team: list) -> int:
+    sum_rating = 0
+    for player in team:
+        sum_rating += player.get_rating()
+    return sum_rating
+
+
+def elves_concert(list_elves: list) -> list[Any]:
+    elves = []
+    for elf in list_elves:
+        elves.append(elf.play_elf_song())
+    return elves
```

Comment:
what is that? Can't we loop over the list of the elves and invoke the method play_elf_song without any additional list?